keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -390,6 +383,10 @@ a single connnection by prepending a stream identififer byte to the data message
 For example, a `stdout` data message sent over the connection will have the `stdout`
 file descriptor (1) prepended to the data message.
 
+### Background: `PortForward` Subprotocol
+
+- TBD
```

Comment:
either drop this section or clarify how resolution of this fits into the alpha timeline (required before alpha, required during alpha, etc)

Reply:
Dropped.